tests/ident/test_engine_parser_msgfplus_2021_03_22.py

```diff
@@ -225,99 +176,14 @@ def test_engine_parsers_msgfplus_internal_next():
             "omssa_mod_dir": Path(__file__).parent.parent / "data",
         },
     )
-    for row in parser._next():
-        assert isinstance(row, dict)
-        assert (
-            row["Raw data location"]
-            == "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/BSA1.mgf"
-        )
-        assert row["Peptide"] == "YICDNQDTISSK"
-        assert row["Modifications"] == "Carbamidomethyl:3"
-        assert row["MS-GF:RawScore"] == "40"
-        assert row["MS-GF:NumMatchedMainIons"] == "3"
-        # search engine not set here, just stuff directly from the mzid
-        break
-
-
-def test_engine_parsers_msgfplus_next():
-    input_file = Path(__file__).parent.parent / "data" / "BSA1_msgfplus_2021_03_22.mzid"
-    rt_lookup_path = Path(__file__).parent.parent / "data" / "BSA1_ursgal_lookup.csv.bz2"
-    db_path = (
-        Path(__file__).parent.parent / "data" / "test_Creinhardtii_target_decoy.fasta"
-    )
-
-    parser = MSGFPlus_2021_03_22(
-        input_file,
-        params={
-            "rt_pickle_name": rt_lookup_path,
-            "database": db_path,
-            "modifications": [
-                {
-                    "aa": "M",
-                    "type": "opt",
-                    "position": "any",
-                    "name": "Oxidation",
-                },
-                {
-                    "aa": "C",
-                    "type": "fix",
-                    "position": "any",
-                    "name": "Carbamidomethyl",
-                },
-                {
-                    "aa": "*",
-                    "type": "opt",
-                    "position": "Prot-N-term",
-                    "name": "Acetyl",
-                },
-            ],
-            "omssa_mod_dir": Path(__file__).parent.parent / "data",
-        },
+    df = parser.unify()
+    row = df.iloc[0, :]
+    assert (
+        row["Raw data location"]
+        == "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/BSA1.mzML"
     )
-    row = next(parser)
-    assert isinstance(row, UnifiedRow)
     assert row["Sequence"] == "YICDNQDTISSK"
     assert row["Modifications"] == "Carbamidomethyl:3"
     assert row["MS-GF:RawScore"] == "40"
-    assert row["MS-GF:NumMatchedMainIons"] == "3"
+    assert row["MS-GF:Num Matched Ions"] == "3"
     assert row["Search Engine"] == "msgfplus_2021_03_22"
```

Comment:
same as above - please also assert identity of the resulting df - not just first line :)